crawl-ref/source/initfile.cc

```diff
@@ -393,6 +393,8 @@ const vector<GameOption*> game_options::build_options_list()
         new StringGameOption(SIMPLE_NAME(tile_font_msg_family), "monospace"),
         new StringGameOption(SIMPLE_NAME(tile_font_stat_family), "monospace"),
         new StringGameOption(SIMPLE_NAME(tile_font_lbl_family), "monospace"),
+        new StringGameOption(SIMPLE_NAME(glyph_mode_font), "monospace"),
+        new IntGameOption(SIMPLE_NAME(glyph_mode_font_size), 24, 0, INT_MAX),
```

Comment:
Saner bounds checking here for reasonable font sizes should be done. A value for this option anywhere close to INT_MAX is certainly a user error.